crates/key-server/src/server.rs

```diff
@@ -438,6 +440,28 @@ impl Server {
         )
         .await
     }
+
+    /// Spawn a metrics push background jobs that push metrics to seal-proxy
+    fn spawn_metrics_push_job(
+        &self,
+        registry: prometheus::Registry,
+    ) -> JoinHandle<()> {
+        let bearer_token = seal_proxy::var!("METRICS_PROXY_BEARER_TOKEN");
```

Comment:
a bearer token is mandatory if the proxy is used, no?
if so, let's check below that it's not empty